parser.py

```diff
@@ -86,6 +92,32 @@ def tweet_json_to_markdown(tweet, username, archive_media_folder, output_media_f
     body += f'\n\n(Originally on Twitter: [{timestamp_str}](https://twitter.com/{username}/status/{tweet_id_str}))'
     return timestamp, body
 
+def is_short_url(url):
+    hostname = urlparse(url).hostname
+    shorteners = ['t.co', '7ax.de', 'bit.ly', 'buff.ly', 'cnn.it', 'ct.de', 'flic.kr', 'go.shr.lc', 'ift.tt', 'instagr.am', 'is.gd', 'j.mp', 'ku-rz.de', 'p.dw.com', 'pl0p.de', 'spon.de', 'sz.de', 'tiny.cc', 'tinyurl.com', 'trib.al', 'wp.me', 'www.sz.de', 'yfrog.com']

```

Comment:
v.gd is another one ("partner" of is.gd) and https://goo.gl/ (Google's own) would be two major ones I feel are missing.